pages/common/dotnet-build.md

```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Builds a .NET application and its dependencies.
+> More information: <https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build>.
```

Comment:
I was just thinking for these URLs across all the Dotnet PRs you've created (and I guess any existing ones?) it might be worth stripping the `en-us` slug. That way it should automatically use the locale that the user uses. 👍🏻

```suggestion
> More information: <https://docs.microsoft.com/dotnet/core/tools/dotnet-build>.
```